util/envDynawo.sh

```diff
@@ -879,16 +880,16 @@ build_dynawo_core() {
   fi
   if [ "$DYNAWO_CMAKE_GENERATOR" = "Unix Makefiles" ]; then
     cd $DYNAWO_BUILD_DIR
-    make -j $DYNAWO_NB_PROCESSORS_USED && make -j $DYNAWO_NB_PROCESSORS_USED install
+    make -j $DYNAWO_NB_PROCESSORS_USED install
   else
-    cmake --build $DYNAWO_BUILD_DIR $DYNAWO_CMAKE_BUILD_OPTION --config $DYNAWO_BUILD_TYPE && cmake --build $DYNAWO_BUILD_DIR --target install --config $DYNAWO_BUILD_TYPE
+    cmake --build $DYNAWO_BUILD_DIR $DYNAWO_CMAKE_BUILD_OPTION --target install
   fi
   RETURN_CODE=$?
   return ${RETURN_CODE}
 }
 
-# Compile a Dynawo library, use help to see all targets
-build_dynawo_lib() {
+# Compile a Dynawo target, use help to see all targets
+build_dynawo_target() {
```

Comment:
Just to be sure: we can still build a single model?

Reply:
yes. I just rename the function because it builds a cmake target and not a lib (just a matter of semantic)